tests/Resources/Page/OCPDashboard/Builds/Builds.robot

```diff
@@ -37,4 +37,42 @@ Check Image Build Status
   Run Keyword If    "${build_status}" == "${target_status}"
   ...    Log   The '${build_name}' image match the expected target status
   ...  ELSE
-  ...    Fail  The '${build_name}' image build status doesn't match the target status '${target_status}'
\ No newline at end of file
+  ...    Fail  The '${build_name}' image build status doesn't match the target status '${target_status}'
+
+Search Last Build
+    [Documentation]    Returns last build
+    [Arguments]    ${namespace}    ${word-to-search-build}
```

Comment:
What about ${build-name-includes}

Reply:
Please, can you write in the documentation what happens if there are two builds matching ${build-name-includes}? Is it really giving you the latest one?

Reply:
as alternative, there is `Search Last Item Instance By Title in OpenShift Table` keywords which search in an openshift table and sort the table by creation date (DESC order)

Reply:
Done,
Now it returns the latest build,I have sorted using the creation time of build.